src/controllers/monitorController.js

```diff
@@ -0,0 +1,35 @@
+import dbQuery from '../db/config.js';
+import { nanoid } from 'nanoid';
+import { queryGetAllMonitors, queryAddMonitor } from '../db/queries.js';
+
+const getMonitors = async (req, res) => {
+  try {
+    const response = await dbQuery(queryGetAllMonitors);
+    const monitors = response.rows;
+    res.json(monitors);
+  } catch (error) {
+    console.error(error);
+    res.status(500).json({ error: 'Could not fetch monitors.' });
+  }
+};
+
+const addMonitor = async (req, res) => {
+  const { schedule } = req.body;
+  const endpoint_id = nanoid(10);
+  try {
+    const response = await dbQuery(queryAddMonitor, [endpoint_id, schedule]);
+    const monitor = response.rows;
```

Comment:
I have a slight preference for this to be response.rows[0] - just to return a single object rather than an array of length 1.